scripts/installer/install.sh

```diff
@@ -283,20 +273,39 @@ platform_install_bootloader_entry()
 
 onie_platform=$(onie-sysinfo -p)
 
-# <arch>-<vendor>-<platform>-<revision>
```

Comment:
This comment didn't survive the code move. Was that intentional?

Reply:
Sort of. as it is a bit misleading. It implies the difference is whether the parts are separated by dash or underscore, but that's not actually true. Both ONIE and ONL use between parts; the difference is wether within a part dashes or underscores are used.

Reply:
I'll readd it and fix it up in a later MR